packages/common/src/ui/layout/tables/components/Header/Header.tsx

```diff
@@ -50,12 +50,30 @@ export const HeaderCell = <T extends RecordWithId>({
     150
   );
 
-  const tooltip = (
+  const showTooltip = !!description || sortable;
+  const tooltip = showTooltip ? (
```

Comment:
simple way to not display Tooltips - they never show for zero length strings

Reply:
Nice!